arch/Mips/MipsInstPrinter.c

```diff
@@ -121,6 +121,106 @@ static void printRegName(MCInst *MI, SStream *OS, MCRegister Reg)
 	SStream_concat0(OS, Mips_LLVM_getRegisterName(Reg, syntax_opt & CS_OPT_SYNTAX_NOREGNAME));
 }
 
+static void patch_cs_printer(MCInst *MI, SStream *O) {
+	if (MI->csh->syntax & CS_OPT_SYNTAX_NO_DOLLAR) {
+		char *dollar = strchr(O->buffer, '$');
+		if (!dollar) {
+			return;
```

Comment:
The `#` is not patched out in this case?

Reply:
I fixed this. https://github.com/capstone-engine/capstone/pull/2665/files#diff-5005d8a75cd7c022055141a16d6d79ead3bec5bf5b8187742a23ce2e7ccad67eR126